keyboards/donutcables/budget96/keymaps/via/keymap.c

```diff
@@ -0,0 +1,46 @@
+/* Copyright 2018 MechMerlin
```

Comment:
```suggestion
/* Copyright 2020 MechMerlin
```
I think this should be updated, if you want to.